.github/workflows/build.yaml

```
@@ -35,19 +38,31 @@ jobs:
+        uses: docker/setup-qemu-action@v3
         with:
-          token: ${{ secrets.CODECOV }}
-          files: ./coverage/*.out
-          verbose: true
+          platforms: all
+          cache-image: true
 
       - name: Set up Docker Build
         uses: docker/setup-buildx-action@v3
+        with:
+          name: supermq-qemu
```

actionlint (1.7.7)

input "name" is not defined in action "docker/setup-buildx-action@v3". available inputs are "append", "buildkitd-config", "buildkitd-config-inline", "buildkitd-flags", "cache-binary", "cleanup", "config", "config-inline", "driver", "driver-opts", "endpoint", "install", "platforms", "use", "version"

(action)